app/src/main/java/de/danoeh/antennapod/fragment/preferences/PlaybackStatisticsFragment.java

```diff
@@ -156,6 +157,8 @@ private void doResetStatistics() {
             disposable.dispose();
         }
 
+        UserPreferences.resetUsageCounting();
```

Comment:
As a safeguard, could you please move this into the `.subscribe` lambda? If for whatever reason the database call fails, the date should not be reset. I do not think this will ever happen (or if it happens, users will not care) but it feels somehow more clean to only update after it is actually reset.